AngryMouse/App.xaml.cs

```diff
@@ -114,7 +115,30 @@ private void CreateContextMenu()
                     _aboutWindow.Closed += (sender, args) => { _aboutWindow = null; };
                 }
             };
+            
+            menu.Items.Add("Run at Windows startup").Click += (s, e) =>
+            {
+                ToolStripMenuItem runStartup = (ToolStripMenuItem)s;
+                runStartup.Checked = !runStartup.Checked;
+                if (runStartup.Checked)
+                {
+                    RunOnStartup.setRunOnStartup(true);
+                }
+                else
+                {
+                    RunOnStartup.setRunOnStartup(false);
+                }
+            };
+            foreach(ToolStripItem item in menu.Items)
```

Comment:
I would just do something like this to avoid having to loop through the items:
```csharp
var startUpItem = new ToolStripMenuItem("Run at Windows startup");
startUpItem.Checked = RunOnStartup.isRunOnStartup();
startUpItem.Click = ...;
menu.Items.Add(startUpItem);
```